iroh-base/Cargo.toml

```diff
@@ -21,6 +21,7 @@ data-encoding = { version = "2.3.3", optional = true }
 hex = "0.4.3"
 multibase = { version = "0.9.1", optional = true }
 postcard = { version = "1", default-features = false, features = ["alloc", "use-std", "experimental-derive"], optional = true }
+redb = { version = "1.5.0", optional = true }
```

Comment:
please add an explicit feature

```toml
redb = ["dep:redb"]
```

Reply:
side note, we should update all our feature definitions to use `dep:` for dependencies

Reply:
Didn't know you can do this. This is cool. Was a bit weirded out by this whole "dependencies are features" thing.